src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/ProcessorTests.cs

```diff
@@ -62,12 +67,17 @@ public static IEnumerable<object[]> GetValidInputsWithProcessSettings()
             yield return new object[]
             {
                 new Hl7v2Processor(null), new Hl7v2Processor(new ProcessorSettings()), new Hl7v2Processor(positiveTimeOutSettings), new Hl7v2Processor(negativeTimeOutSettings),
-                new Hl7v2TemplateProvider("TestTemplates"), _hl7v2TestData,
+                new Hl7v2TemplateProvider("TestTemplates"), _hl7v2TestData
```

Comment:
How about using a constant string since it appears many times?

Reply:
Sure. A constant is added to replace the raw strings.